test/StakeManager.js

```diff
@@ -1187,40 +1187,6 @@ describe('StakeManager', function () {
       await stakeManager.connect(signers[0]).unpause();
     });
 
-    it('Delegation should revert, if staker is inactive for more than grace period', async function () {
```

Comment:
@SamAg19 removed this test, since we don't have `_isStakerActive` check anymore. Do we still need some level of testing for this?

Reply:
No i don't think so it necessary

Reply:
Resolved